stable/traefik/README.md

```diff
@@ -119,6 +119,7 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `accessLogs.enabled`            | Whether to enable Traefik's access logs                              | `false`                                   |
 | `accessLogs.filePath`           | The path to the log file. Logs to stdout if omitted                  | None                                      |
 | `accessLogs.format`             | What format the log entries should be in. Either `common` or `json`  | `common`                                  |
+| `gcp.staticIP`            | An available IP you have reserved on Google Cloud Platform           | None                                      |
```

Comment:
What do you think of @krancour's suggestion? As I mentioned in my comment im not sure how this works with other clouds. But you know that this works with more than gcp?